### PR TITLE
Use the same UTC time zone across the library.

### PR DESCRIPTION
**Pull request by lolepezy**
This is to make sure we use only UTC everywhere.

### CODE REVIEW

File: src/main/java/net/ripe/rpki/commons/util/UTC.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+public class UTC {
+    public static DateTime dateTime() {
+        return new DateTime(DateTimeZone.UTC);
+    }
+
+    public static DateTime dateTime(Object o) {
+        return new DateTime(o, DateTimeZone.UTC);
+    }
+
+    public static Instant instant() {
```
Comment (ties) [inline src/main/java/net/ripe/rpki/commons/util/UTC.java]: Instant is in UTC 👍

File: src/main/java/net/ripe/rpki/commons/util/UTC.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+package net.ripe.rpki.commons.util;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import java.time.Instant;
+
+public class UTC {
+    public static DateTime dateTime() {
```
Comment (ties) [inline src/main/java/net/ripe/rpki/commons/util/UTC.java]: A different name (`utcNow`) + static imports may give slightly more fluent code.

On the other hand, I like that getting an instance makes sure that the clock does not change between calls (so two years is two years and not potentially two years minus one second)

Comment (lolepezy) [reply src/main/java/net/ripe/rpki/commons/util/UTC.java]: I like it being UTC.dateTime(),  UTC.dateTime(whatever) and UTC.instant(). Looks very "in your face" UTC.

Comment (ties) [reply src/main/java/net/ripe/rpki/commons/util/UTC.java]: True – and it has a nice symmetry with `new DateTime(..., DateTimeZone.UTC)`.

**Pull request merged**